--- a/src/beman/exemplar/CMakeLists.txt	2025-03-05 06:01:47.000000000 -0500
+++ b/src/beman/exemplar/CMakeLists.txt	2025-05-09 19:34:26.861986882 -0400
@@ -1,13 +1,12 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

-add_library(beman.exemplar)
+add_library(beman.exemplar INTERFACE)
 add_library(beman::exemplar ALIAS beman.exemplar)

-target_sources(beman.exemplar PRIVATE identity.cpp)

 target_sources(
     beman.exemplar
-    PUBLIC
+    INTERFACE
         FILE_SET HEADERS
         BASE_DIRS ${PROJECT_SOURCE_DIR}/include
         FILES ${PROJECT_SOURCE_DIR}/include/beman/exemplar/identity.hpp
@@ -21,9 +20,6 @@
 install(
     TARGETS beman.exemplar
     EXPORT beman.exemplar
-    DESTINATION
-    ${CMAKE_INSTALL_LIBDIR}$<$<CONFIG:Debug>:/debug>
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}$<$<CONFIG:Debug>:/debug>
     FILE_SET HEADERS DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
 )

@@ -38,7 +34,7 @@
     )

     write_basic_package_version_file(
-        "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}/${PROJECT_NAME}-version.cmake"
+        "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}/${PROJECT_NAME}-config-version.cmake"
         VERSION ${PROJECT_VERSION}
         COMPATIBILITY SameMajorVersion
     )
@@ -46,7 +42,7 @@
     install(
         FILES
             "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}/${PROJECT_NAME}-config.cmake"
-            "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}/${PROJECT_NAME}-version.cmake"
+            "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}/${PROJECT_NAME}-config-version.cmake"
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
         COMPONENT development
     )
