--- a/src/beman/exemplar/CMakeLists.txt	2025-03-05 06:01:47.000000000 -0500
+++ b/src/beman/exemplar/CMakeLists.txt	2025-05-09 19:34:26.861986882 -0400
@@ -2,14 +2,13 @@

 include(GNUInstallDirs)

-add_library(beman.exemplar)
+add_library(beman.exemplar INTERFACE)
 add_library(beman::exemplar ALIAS beman.exemplar)

-target_sources(beman.exemplar PRIVATE identity.cpp)

 target_sources(
     beman.exemplar
-    PUBLIC
+    INTERFACE
         FILE_SET HEADERS
         BASE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/../../../include
         FILES
@@ -24,9 +23,6 @@
 install(
     TARGETS beman.exemplar COMPONENT beman.exemplar
     EXPORT beman.exemplar
-    DESTINATION
-    ${CMAKE_INSTALL_LIBDIR}$<$<CONFIG:Debug>:/debug>
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}$<$<CONFIG:Debug>:/debug>
     FILE_SET HEADERS DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
 )

@@ -41,7 +37,7 @@
     )

     write_basic_package_version_file(
-        "${CMAKE_CURRENT_BINARY_DIR}/beman.exemplar-version.cmake"
+        "${CMAKE_CURRENT_BINARY_DIR}/beman.exemplar-config-version.cmake"
         VERSION ${PROJECT_VERSION}
         COMPATIBILITY ExactVersion
     )
@@ -49,7 +45,7 @@
     install(
         FILES
             "${CMAKE_CURRENT_BINARY_DIR}/beman.exemplar-config.cmake"
-            "${CMAKE_CURRENT_BINARY_DIR}/beman.exemplar-version.cmake"
+            "${CMAKE_CURRENT_BINARY_DIR}/beman.exemplar-config-version.cmake"
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/beman.exemplar"
         COMPONENT beman.exemplar
     )
